website/src/graphql/dailp/queries.graphql

```diff
@@ -427,48 +427,39 @@ query BookmarkedDocuments {
   }
 }
 
+fragment CommentFields on Comment {
+  id
+  postedAt {
+    timestamp
+    date {
+      year
+      month
+      day
+      formattedDate
+    }
+  }
+  postedBy {
+    id
+    displayName
+  }
+  textContent
+  commentType
+}
```

Comment:
These common fields used by two queries and the post mutation are now in a fragment.